docs/book/src/clusteropenstack/configuration.md

```diff
@@ -654,6 +656,16 @@ spec:
 
 If `managedSecurityGroups: true`, security group rule opening 22/tcp is added to security groups for bastion, controller, and worker nodes respectively. Otherwise, you have to add `securityGroups` to the `bastion` in `OpenStackCluster` spec and `OpenStackMachineTemplate` spec template respectively.
 
+### Making changes to the bastion host
+
+Changes can be made to the bastion instance, like for example changing the flavor.
+First, you have to disable the bastion host by setting `enabled: false` in the `OpenStackCluster.Spec.Bastion` field. Then, you can make changes to the instance spec and re-enable the bastion host by setting `enabled: true` and by modifying the `OpenStackCluster.Spec.Bastion.Instance` field.
+
+### Disabling the bastion
+
+To disable the bastion host, set `enabled: false` in the `OpenStackCluster.Spec.Bastion` field. The bastion host will be deleted.
+Then you can remove the `OpenStackCluster.Spec.Bastion` field from the `OpenStackCluster` spec.
```

Comment:
line 667 seems confusing, what need to check or something to be done or how long should we wait?
that I can remove `OpenStackCluster.Spec.Bastion` field ? if it's disabled, why need remove them or what's the right timing to do so?

Reply:
I'll add more content.